GitSearch/Application/SearchResult/SearchResultViewController.swift

```diff
@@ -33,7 +33,7 @@ class SearchResultViewController: UICollectionViewController, UICollectionViewDe
     }
     
     override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        guard let count = gitRep?.items?.count else { return 0 }
+        guard let count = gitRep.items?.count else { return 0 }
         return count
```

Comment:
In this case, you can use this swift idiom: 
```
    return gitRep.items?.count ?? 0
```

Reply:
thank you, done